source/com.google.android.material/material/Transforms/Metadata.xml

```diff
@@ -218,5 +218,41 @@
     <remove-node
         path="/api/package[@name='com.google.android.material.snackbar']/class[@name='Snackbar.SnackbarLayout']/method[@name='setLayoutParams' and count(parameter)=1 and parameter[1][@type='android.view.ViewGroup.LayoutParams']]"
         />
+ 
+        
+    <attr
+        path="/api/package[@name='com.google.android.material.internal']/class[@name='RectEvaluator']/method[@name='evaluate' and count(parameter)=3 and parameter[1][@type='float'] and parameter[2][@type='android.graphics.Rect'] and parameter[3][@type='android.graphics.Rect']]/parameter[1]"
+        name="managedType"
+        >
+        float
+      </attr>
+      <attr
+      path="/api/package[@name='com.google.android.material.internal']/class[@name='RectEvaluator']/method[@name='evaluate' and count(parameter)=3 and parameter[1][@type='float'] and parameter[2][@type='android.graphics.Rect'] and parameter[3][@type='android.graphics.Rect']]/parameter[2]"
+      name="managedType"
+      >
+      Java.Lang.Object?
+    </attr>
+    <attr
+      path="/api/package[@name='com.google.android.material.internal']/class[@name='RectEvaluator']/method[@name='evaluate' and count(parameter)=3 and parameter[1][@type='float'] and parameter[2][@type='android.graphics.Rect'] and parameter[3][@type='android.graphics.Rect']]/parameter[3]"
+      name="managedType"
+      >
+      Java.Lang.Object?
+    </attr>
+    <attr
+      path="/api/package[@name='com.google.android.material.internal']/class[@name='RectEvaluator']/method[@name='evaluate' and count(parameter)=3 and parameter[1][@type='float'] and parameter[2][@type='android.graphics.Rect'] and parameter[3][@type='android.graphics.Rect']]"
+      name="managedReturn"
+      >
+      Java.Lang.Object
+    </attr>
+
+
+    <!-- 
+      error CS0234: The type or namespace name 'ISheet' does not exist in the namespace 'Google.Android.Material.SideSheet' 
+
+      ISheet - visibility=""
+    -->
+    <remove-node 
```

Comment:
We should try to avoid adding large amounts of copied auto-generated additions.  This code will never get updated as we make improvements to `generator`.

Is it possible to fix this by removing the `<implements name="Sheet">` node or by making the `Sheet` interface public?

Reply:
let me try

Reply:
Fixed by removing `<implements name="Sheet">`

I was told not to sruface default package visibilty (`visibility=""`) because of ACW generation. There is even utility step that checks that metadata does not contain code that does so.